Library/Koala/KoalaTrackingClient.swift

```diff
@@ -57,7 +57,7 @@ public final class KoalaTrackingClient: TrackingClientType {
     print("🐨 [Koala Track]: \(event), properties: \(properties)")
 
     self.queue.async {
-      BITHockeyManager.shared().metricsManager.trackEvent(withName: event)
+      Answers.logCustomEvent(withName: event, customAttributes: properties)
```

Comment:
While we _can_ pass `properties` through here, it might not be necessary and we might want to reduce the payload?

Reply:
Nice, @justinswart ! Yeah, we can see if the logs are enough without the properties and simply add them in the future in case we need it.

Reply:
Ok cool, will pass `nil` for properties for now.